api/post.go

```diff
@@ -475,7 +478,7 @@ func getMentionKeywordsInChannel(profiles map[string]*model.User) map[string][]s
 		}
 
 		// Add @channel and @all to keywords if user has them turned on
-		if int64(len(profiles)) < *utils.Cfg.TeamSettings.MaxNotificationsPerChannel && profile.NotifyProps["channel"] == "true" {
+		if profile.NotifyProps["channel"] == "true" {
```

Comment:
Did you purposely remove the check of `*utils.Cfg.TeamSettings.MaxNotificationsPerChannel` here?